### ISSUE TITLE
Support apiPrefix from confluence.js

### ISSUE THREAD

**Issue by ThePlenkov** [markdown-confluence/markdown-confluence#350]
Hi!

We recently added a new parameter to confluence.js `apiPrefix` allowing us to use custom prefix instead of default /wiki/rest. Could you please also proxy it via your json config?

Thanks!

**Comment by andymac4182** [comment]
Yes. Is this targeted at on-premise support or [custom domain support](https://jira.atlassian.com/browse/CLOUD-6999)?

**Comment by ThePlenkov** [comment]
Hi! Yes - it's to support our custom domain, we have just /rest instead of /wiki/rest

**Comment by andymac4182** [comment]
Thanks. Will review the pr.

**Comment by michellepolte-optimizely** [comment]
Hey, what's going on with the support of apiPrefix option?

**Comment by alxwest** [comment]
This would be nice to have supported.
